src/Jackett.Common/Definitions/letseed.yml

```diff
@@ -0,0 +1,240 @@
+---
+id: letseed
+name: LetSeed
+description: "LetSeed is a TURKISH Private Torrent Tracker for MOVIES / TV / GENERAL"
+# The HTML says `lang="en-US"` but the web UI defaults to Turkish, the categories are
+# always in Turkish, and there are more UI elements when using the Turkish UI:
+language: tr-TR
+type: private
+encoding: UTF-8
+links:
+  - https://letseed.org/
+
+caps:
+  categorymappings:
+    - {id: 3, cat: TV, desc: "Diziler"}
+    - {id: 5, cat: TV/Foreign, desc: "Yabancı Diziler"}
+    - {id: 6, cat: TV/Other, desc: "Yerli Dizi Boxset"}
+    - {id: 4, cat: TV/Other, desc: "Yerli Diziler"}
+    - {id: 44, cat: Books, desc: "Eğitim"}
+    - {id: 46, cat: Books/EBook, desc: "E-Book"}
+    - {id: 45, cat: Books/Other, desc: "Eğitim"}
+    - {id: 1, cat: Movies, desc: "Filmler"}
+    - {id: 15, cat: Movies/UHD, desc: "4K"}
+    - {id: 2, cat: Movies/Other, desc: "Animasyonlar"}
+    - {id: 16, cat: Movies/Other, desc: "BoxSet"}
+    - {id: 14, cat: Movies/Foreign, desc: "Yabancı Yapımlar"}
+    - {id: 13, cat: Movies/Other, desc: "Yerli Yapımlar"}
+    - {id: 32, cat: Audio, desc: "Müzik"}
+    - {id: 35, cat: Audio/Other, desc: "Diskografi"}
+    - {id: 34, cat: Audio/Foreign, desc: "Yabancı"}
+    - {id: 33, cat: Audio/Other, desc: "Yerli"}
+    - {id: 28, cat: PC/Games, desc: "Oyun"}
+    - {id: 29, cat: PC/Games, desc: "PC"}
+    - {id: 31, cat: Console/PS4, desc: "Playstation"}
+    - {id: 30, cat: Console/XBox, desc: "XBOX"}
+    - {id: 19, cat: Other, desc: "Platform Sunumları"}
+    - {id: 21, cat: Other, desc: "AMZN"}
+    - {id: 23, cat: Other, desc: "BluTv"}
+    - {id: 22, cat: Other, desc: "DSNP"}
+    - {id: 24, cat: Other, desc: "Exxen"}
+    - {id: 26, cat: Other, desc: "GAIN"}
+    - {id: 20, cat: Other, desc: "Netflix"}
+    - {id: 42, cat: Other, desc: "Tabii"}
+    - {id: 25, cat: Other, desc: "TOD"}
+    - {id: 27, cat: Other, desc: "TV+"}
+    - {id: 36, cat: PC, desc: "Program"}
+    - {id: 37, cat: PC, desc: "İşletim Sistemi"}
+    - {id: 38, cat: PC/Mac, desc: "Mac"}
+    - {id: 39, cat: PC, desc: "Windows"}
+    - {id: 40, cat: TV/Sport, desc: "Spor"}
+    - {id: 41, cat: TV/Sport, desc: "Spor Sunumları"}
+    - {id: 17, cat: Other, desc: "VIP Sunumlar"}
+    - {id: 18, cat: Other, desc: "VIP Sunumlar"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Filter freeleech only
```

Comment:
```suggestion
    label: Search freeleech only
```

Reply:
Ah kk, forgot that this site engine orders results by freeleech and not, rather than only showing freeleech. Ignore this.